<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurnPageGridView</class>
 <widget class="QWidget" name="TurnPageGridView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="lastPageBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	image: url(:/icons/leftLittleArrow.png);
	border: 0px;
}

QPushButton:hover
{
	background-color: rgb(202, 202, 202);
}

QPushButton:pressed
{
	background-color: gray;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextPageBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	image: url(:/icons/rightLittleArrow.png);
	border: 0px;
}

QPushButton:hover
{
	background-color: rgb(202, 202, 202);
}

QPushButton:pressed
{
	background-color: gray;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
